build/configs/imxrt1020-evk/loadable_elf_apps/Make.defs

```diff
@@ -168,6 +168,6 @@ HOSTCFLAGS = -Wall -Wstrict-prototypes -Wshadow -Wundef -g -pipe
 HOSTLDFLAGS =
 
 define DOWNLOAD
-  $(TOPDIR)/../build/configs/$(CONFIG_ARCH_BOARD)/$(CONFIG_ARCH_BOARD)_download.sh $(1) $(2) $(3) $(4) $(5) $(6)
+  $(TOPDIR)/../build/configs/common_download.sh $(1) $(2) $(3) $(4) $(5) $(6)
```

Comment:
I think we can move this script calling into here.
https://github.com/Samsung/TizenRT/blob/master/os/Makefile.unix#L546 
What do you think about this?

Reply:
Okay, I will move the common download script calling to common makefile.

Reply:
The common download script is currently not supported by all the boards. So moving the script calling to the common makefile will not be supported by the other boards(as they don't support common download). Please let me know your thoughts regarding this.

Reply:
Ok. Then let's move after applying to other boards as common download.